pkg/confluence/api.go

```diff
@@ -444,15 +476,16 @@ func (api *API) UpdatePage(
 
 	if page.Type != "blogpost" {
 		if len(page.Ancestors) == 0 {
-			return fmt.Errorf(
-				"page %q info does not contain any information about parents",
-				page.ID,
-			)
-		}
-
-		// picking only the last one, which is required by confluence
-		oldAncestors = []map[string]interface{}{
-			{"id": page.Ancestors[len(page.Ancestors)-1].Id},
+			oldAncestors = nil
```

Comment:
No need to set it to nil here since you can just make an opposition of the if-condition, like this:

```go
if page.Type != "blogpost" && len(page.Ancestors) > 0 {
	oldAncestors = []map[string]interface{}{
		{"id": page.Ancestors[len(page.Ancestors)-1].Id},
	}
}
```